vita3k/modules/SceAppMgr/SceAppMgr.h

```diff
@@ -53,8 +54,15 @@ typedef struct SceAppMgrLoadExecOptParam {
     int reserved[256 / 4]; //!< Reserved area
 } SceAppMgrLoadExecOptParam;
 
+typedef struct sceAppMgrAppParamGetStringOptParam { // size is 0x10 on 3.60
```

Comment:
This comment is a bit misleading, it makes you think you are talking about the size field of sceAppMgrAppParamGetStringOptParam and not the size of the structure.

Reply:
Is @bookmist  comment

Reply:
Don't use size with a field named size right below, you can use anything else (like this structure is expected to be 0x10 bytes on 3.60 or whatever)

Reply:
Adcorded with bookmist, i have just removed comment.